config/components/webhook/service.yaml

```diff
@@ -8,5 +8,3 @@ spec:
     - port: 443
       protocol: TCP
       targetPort: 9443
-  selector:
```

Comment:
AFAIK the Service needs a selector, right?

Reply:
It's added by Kustomize globally with `commonLabels`.